tutorials/circuits/3_summary_of_quantum_operations.ipynb

```diff
@@ -124,7 +124,8 @@
     "## Single-Qubit Gates <a name=\"single_gates\"/>\n",
```

Comment:
<p>Thanks for adding this! There is just a slight difference between this matrix and the one in UGate API documentation: https://qiskit.org/documentation/stubs/qiskit.circuit.library.UGate.html?highlight=ugate#qiskit.circuit.library.UGate In there the phi and lambda are put inside a braket in the last matrix element. Can you copy the code here to make sure it's exactly the same? Thank you. https://github.com/Qiskit/qiskit-terra/blob/2e64b98e6d8cd60c69ab4179f9dfe4e5cc6d4466/qiskit/circuit/library/standard_gates/u.py#L49-L57</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Corrected. There was an issue with the /newcommand call so I did write it out fully instead. Also there was a second instance in the beginning paragraphs with the same discrepancy, so updated the paragraph to ensure consistency across the document.